service/matching/matchingEngine.go

```diff
@@ -167,6 +169,8 @@ func (e *matchingEngineImpl) String() string {
 // if successful creates one.
 func (e *matchingEngineImpl) getTaskListManager(taskList *taskListID,
 	taskListKind *types.TaskListKind) (taskListManager, error) {
+	// Cache the task list in taskListCache
+	e.cacheTaskListByDomain(taskList)
```

Comment:
Only cache user defined task lists? Other tasklists like sticky tasklist should not be cached I think?

Reply:
How can i differentiate user defined and stiky tasklist?

Reply:
Check the `isRawListList()` method in frontend.